### PR TITLE
Add sic package

### PR DESCRIPTION
**Pull request by r7st**
Adding sic package - simple irc client.

"sic is an extremely simple IRC client. It consists of less than 250 lines of code. It is the little brother of irc it."
https://tools.suckless.org/sic/

### CODE REVIEW

**Comment by oetiker** [comment]
thanks for your contribution ... I guess @hadfl or @citrus-it should say something about their willingness to takes this into the repo

**Review by hadfl — COMMENTED** [review]
Thank you for your contribution. This looks pretty good. I just added a few comments for changes and hope they make sense.

File: build/sic/build.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+build64() {
+    pushd $TMPDIR/$BUILDDIR > /dev/null
+    logmsg "Building 64-bit"
+    make_clean
+    make_prog64
+    [ -z "$SKIP_BUILD_ERRCHK" ] && check_buildlog ${EXPECTED_BUILD_ERRS:-0}
+    make_install64
+    popd > /dev/null
+}
```
[the remark this answers is not in the document]
Comment (r7st) [reply build/sic/build.sh]: That's a lot cleaner, thanks.

File: build/sic/build.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+PKG=ooce/network/sic
+SUMMARY="simple irc client"
+DESC="sic is an extremely simple IRC client. It consists of less than "
+DESC+="250 lines of code. It is the little brother of irc it."
+
+set_arch 64
+
+set_mirror "https://dl.suckless.org/"
+set_checksum sha256 "30478fab3ebc75f2eb5d08cbb5b2fedcaf489116e75a2dd7197e3e9c733d65d2"
+set_builddir $PROG-$VER
```
[the remark this answers is not in the document]
Comment (r7st) [reply build/sic/build.sh]: Great, that helps. I've made the update.

File: build/sic/patches/series [hunk 3]
```diff
@@ -0,0 +1 @@
+Makefile.patch
```
[the remark this answers is not in the document]
Comment (r7st) [reply build/sic/patches/series]: I removed the patch.

**Review by hadfl — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Pull request merged**